File: tests/PartialSealed.pas
namespace SGU.Infra.Properties;

interface

type
  [System.Runtime.CompilerServices.CompilerGeneratedAttribute]
  [System.CodeDom.Compiler.GeneratedCodeAttribute('Microsoft.VisualStudio.Editors.SettingsDesigner.SettingsSingleFileGenerator', '10.0.0.0')]
  Settings = partial sealed class(System.Configuration.ApplicationSettingsBase)
  private
    class var defaultInstance: Settings := (System.Configuration.ApplicationSettingsBase.Synchronized(new Settings()) as Settings);
    class method get_Default: Settings;
  public
    class property Default: Settings read get_Default;
  end;

implementation

class method Settings.get_Default: Settings;
begin
  exit(defaultInstance);
end;

end.

File: tests/PartialSealed.cs
namespace SGU.Infra.Properties {
    [System.Runtime.CompilerServices.CompilerGeneratedAttribute]
    [System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.Editors.SettingsDesigner.SettingsSingleFileGenerator", "10.0.0.0")]
    public partial class Settings : System.Configuration.ApplicationSettingsBase {
        // TODO: field defaultInstance: Settings -> declare a field
        public static Settings defaultInstance = System.Configuration.ApplicationSettingsBase.Synchronized(new Settings()) as Settings;
        public Settings Default { get => get_Default(); }
        public static Settings get_Default() {
            return defaultInstance;
        }
    }
}
